projects/control-service/projects/pipelines_control_service/src/main/resources/schema.graphqls

```diff
@@ -70,6 +70,20 @@ type DataJobResources {
     netBandwitdthLimit: Int
 }
 
+type DataJobExecutionDeployment {
```

Comment:
What necessitates this to be a separate model? Is it not confusing to have two deployment models with an almost similar structure - one as a parent of the executions and one as a child?

Reply:
GraphQL doesn't support inheritance and the DataJobDeployment differs from the model defined in Model project.

Reply:
https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/model/apidefs/datajob-api/api.yaml#L1267